package.json

```diff
@@ -15,13 +15,17 @@
     "index.css"
   ],
   "scripts": {
-    "start": "concurrently 'npm run build' 'npm run serve'",
+    "start": "npm run build && concurrently 'npm run watch' 'npm run serve'",
```

Comment:
Nested concurrently use seems interesting. I guess it works though.

Reply:
yup, tested just now )